### DISCUSSION TITLE
Why is @ftx_official account blocked?

### DISCUSSION THREAD

**Discussion by ghost** [deso-protocol/dips#98] [Q&A]
Every day a new exchange lists Clout. That's why we found it logical to invest in some exchanges. But we couldn't login to @ftx_official account from  bitclout.com. There were also issues with several other accounts. What is the reason?

**Comment by tijno** [comment]
Are these your accounts that are blocked ? 

Either way - not sure its  a CIP question - sounds more like something for bitclout.com node support.

**Reply by ghost** [reply]
They do not respond to e-mail addresses. I guess they only answer questions on Github.

I also sent emails about another issue. But nobody answers. I think they are too busy.
Is there someone from the core team I can send a private message to?
